src/components/Gauge.js

```diff
@@ -0,0 +1,263 @@
+    const endAngle = (135 * (Math.PI / 180));
+    const pie = d3.layout.pie().sort(null).padAngle(props.padAngle).endAngle(endAngle);
+    const values = pie(dataSets);
+    const radius = Math.min(props.width, props.height) / 2;
+    const standardArc = d3.svg.arc().outerRadius(radius - props.activeOffset).innerRadius(radius - props.arcWidth);
+    const hoveredArc = d3.svg.arc().outerRadius(radius).innerRadius(radius - props.arcWidth);
+    const baseArc = d3.svg.arc().outerRadius(radius - props.activeOffset).innerRadius(radius - props.arcWidth).startAngle(startAngle).endAngle(endAngle);
+
+    this.setState({
+      baseArc,
+      endAngle,
+      hoveredArc,
+      pie,
+      radius,
+      standardArc,
+      values
+    });
+  },
+
+  _renderArcs () {
+    const firstSegment = -135 * (Math.PI / 180);
+    const endOfFirstSegment = -90 * (Math.PI / 180);
+    const secondSegment = -90 * (Math.PI / 180);
+    const endOfSecondSegment = -45 * (Math.PI / 180);
+    const thirdSegment = -45 * (Math.PI / 180);
+    const endOfThirdSegment = 0;
+    const fourthSegment = 0;
+    const endOfFourthSegment = 45 * (Math.PI / 180);
+    const fifthSegment = 45 * (Math.PI / 180);
+    const endOfFifthSegment = 90 * (Math.PI / 180);
+    const sixthSegment = 90 * (Math.PI / 180);
+    const endOfSixthSegment = 135 * (Math.PI / 180);
+
+    const segments = [
+      { startAngle: firstSegment, endAngle: endOfFirstSegment, padAngle: 0.02 },
+      { startAngle: secondSegment, endAngle: endOfSecondSegment, padAngle: 0.02 },
+      { startAngle: thirdSegment, endAngle: endOfThirdSegment, padAngle: 0.02 },
+      { startAngle: fourthSegment, endAngle: endOfFourthSegment, padAngle: 0.02 },
+      { startAngle: fifthSegment, endAngle: endOfFifthSegment, padAngle: 0.02 },
+      { startAngle: sixthSegment, endAngle: endOfSixthSegment, padAngle: 0.02 }];
+
+    return segments.map((point, i) => {
+      return (
+        <g
+          key={i}
+        >
+          <path
+            className={'arc-' + this.props.id}
+            d={this.state.standardArc(point)}
+            fill={this.props.colors[i]}
+            opacity={this.props.opacity}
+            ref={'arc-' + this.props.id + i}
+          />
+        </g>
+      );
+    });
+  },
+
+  _renderBaseArc () {
+    return (
+      <g>
+        <path d={this.state.baseArc()} fill={this.props.baseArcColor}></path>
+      </g>
+    );
+  },
+
+  _renderDataPoints () {
+    const dataPoints = this.props.dataPoints.map(dataPoint => {
+      return dataPoint.value;
+    });
+
+    return dataPoints.map((dataPoint, index) => {
+      const percentOfTotal = dataPoint / this.props.chartTotal;
+      const endAngle = (percentOfTotal * 270) - 135;
+
+      const dataPointArc = d3.svg.arc()
+        .outerRadius(this.state.radius - this.props.activeOffset)
+        .innerRadius(this.state.radius - this.props.arcWidth)
+        .startAngle(endAngle * (Math.PI / 180))
+        .endAngle(endAngle * (Math.PI / 180));
```

Comment:
I'm seeing 180 used a lot through out the component.  Can we maybe make this a state variable that is set on `getInitialState`?  Less work to maintain down the road.